### DISCUSSION TITLE
Cant Access Copilot Student version

### DISCUSSION THREAD

**Discussion by Masl321** [community/community#141455] [GitHub Education] [answered]
### Hi everyone,

I have the Student developer pack, but I cant access copilot and the website only shows me a "free trial"

**Comment by davevad93** [comment, marked as answer]
Hi @Masl321 ,

If your status is "`Congrats! Your request for @Masl321 was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can't redeem your offer, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

**Reply by Masl321** [reply]
Thank you! turns out I was too blind to find that and tried to jump in straight away

**Reply by davevad93** [reply]
You're welcome @Masl321 , have a nice day! 👍
